src/core/pyproxy.c

```diff
@@ -1225,9 +1232,69 @@ EM_JS_NUM(int, pyproxy_init_js, (), {
 });
 // clang-format on
 
+JsRef
+create_once_proxy(PyObject* obj)
```

Comment:
Could you add a docstring, as it's not clear from the name that the resulting proxy can only be called once.